lib/embulk/output/bigquery/bigquery_client.rb

```diff
@@ -411,6 +411,9 @@ def create_table_if_not_exists(table, dataset: nil, options: nil)
             dataset ||= @dataset
             options ||= {}
             options['time_partitioning'] ||= @task['time_partitioning']
+            if Helper.has_partition_decorator?(table) && @task['range_partitioning'].present?
+              raise Error, "range_partitioning and partition decorator are not supported at the same time"
+            end
```

Comment:
I think this part will be unnecessary if we add this check in the `bigquery.rb`.